nginx-proxy-manager/umbrel-app.yml

```diff
@@ -0,0 +1,47 @@
+manifestVersion: 1
+id: nginx-proxy-manager
+name: Nginx Proxy Manager
+tagline: Expose your services easily and securely
+icon: https://nginxproxymanager.com/logo.svg
```

Comment:
This can be deleted entirely. We will host the icon at https://github.com/getumbrel/umbrel-apps-gallery

Reply:
Ok, will do